src/components/SidePanel/styles.module.scss

```diff
@@ -22,9 +22,15 @@
 }
 
 .content {
+  border-radius: $border-radius;
+  border: 1px solid $light-gray;
```

Comment:
Now that this has a default border, it also needs a default height (for short content). You can see why by looking at the [FullscreenGrid story](http://localhost:9001/?path=/story/fullscreengrid--default).

This seems to fix everything:

```diff
diff --git a/src/components/SidePanel/styles.module.scss b/src/components/SidePanel/styles.module.scss
index f293df6..0920c48 100644
--- a/src/components/SidePanel/styles.module.scss
+++ b/src/components/SidePanel/styles.module.scss
@@ -24,6 +24,7 @@
 .content {
   border-radius: $border-radius;
   border: 1px solid $light-gray;
+  height: 100%;
   overflow-x: auto;
 }
 
```

Speaking of which, the background color on these panels (just for the story) isn't necessary anymore since there's a border. You can take them out:

```diff
diff --git a/src/components/SidePanel/styles.module.scss b/src/components/SidePanel/styles.module.scss
index f293df6..0920c48 100644
--- a/src/components/SidePanel/styles.module.scss
+++ b/src/components/SidePanel/styles.module.scss
@@ -24,6 +24,7 @@
 .content {
   border-radius: $border-radius;
   border: 1px solid $light-gray;
+  height: 100%;
   overflow-x: auto;
 }
 
diff --git a/stories/setup/styles.scss b/stories/setup/styles.scss
index a776fa0..707b2c3 100644
--- a/stories/setup/styles.scss
+++ b/stories/setup/styles.scss
@@ -77,18 +77,10 @@ body.fullscreen {
   padding: 20px;
 }
 
-.FullscreenGridStory-mainSidePanel {
-  background-color: lighten(orange, 44);
-}
-
 .FullscreenGridStory-content {
   background-color: lighten(yellow, 47);
 }
 
-.FullscreenGridStory-altSidePanel {
-  background-color: lighten(green, 68);
-}
-
 .CodeOverview-container {
   display: grid;
   grid-template-columns: 3fr 1fr;
```